selfdrive/car/subaru/values.py

```diff
@@ -105,11 +117,13 @@ class CAR(Platforms):
     "SUBARU OUTBACK 6TH GEN",
     SubaruCarInfo("Subaru Outback 2020-22", "All", car_parts=CarParts.common([CarHarness.subaru_b])),
     specs=CarSpecs(mass=1568, wheelbase=2.67, steerRatio=17),
+    flags=SubaruFlags.GLOBAL_GEN2 | SubaruFlags.STEER_RATE_LIMITED,
```

Comment:
Should we add an `adas_platform` field that is an enum not a flag? You can set GEN2 and PREGLOBAL in one car, and we have no tests to catch this

Reply:
We can just sub-class for this